spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -95,6 +95,16 @@
 
   let(:decrypter) { Aes256CbcEncryptor.new(Settings.sso.cookie_key, Settings.sso.cookie_iv) }
 
+  def verify_session_cookie
+    token = session[:token]
+    expect(token).to_not be_nil
+    session_object = Session.find(token)
+    expect(session_object).to_not be_nil
+    session_object.to_hash.each do |k, v|
+      expect(session[k]).to eq(v)
+    end
+  end
```

Comment:
i think as part of verifying the session token, we also need to make sure to verify the internal expiration of the cookie. A session cookie is going to be good for the entirety of the user's browser experience. We want to be able to destroy the cookie and the session if the session expires, and i think we should track the expiration inside the cookie similar to what we do for sso cookie.

Reply:
@U-DON did we ever introduce this change or are we relying on the session_object timeout for now? Either would work, but I would like to call attention to this since its really difficult to test it and really important to verify its working currently.